C/common/storage_client.cpp

```diff
@@ -19,6 +20,14 @@
 #include <thread>
 #include <map>
 #include <string_utils.h>
+#include <sys/uio.h>
+#include <errno.h>
+
+#define INSTRUMENT	1
```

Comment:
Let's set INSTRUMENT to 0 before checking in

Reply:
done